meta-client/src/main/java/org/apache/helix/metaclient/api/MetaClientReconnectPolicy.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.helix.metaclient.api;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Policy to define client re-establish connection behavior when connection to underlying metadata
+ * store is expired.
+ */
+
+public interface MetaClientReconnectPolicy {
+
+  enum RetryPolicyName {
+    NO_RETRY,
+    EXP_BACKOFF,
+    LINEAR_BACKOFF
```

Comment:
Curious, are we going to implement this one in future PRs?

Reply:
EXP_BACKOFF is implemented in ZKClient. We will reuse.
the other two is scheduled for future.